diff --git a/node_modules/qtumjs-wallet/lib/Insight.js b/node_modules/qtumjs-wallet/lib/Insight.js
index ed689e1..864682b 100644
--- a/node_modules/qtumjs-wallet/lib/Insight.js
+++ b/node_modules/qtumjs-wallet/lib/Insight.js
@@ -15,8 +15,8 @@ const axios_1 = __importDefault(require("axios"));
 const constants_1 = require("./constants");
 const INSIGHT_BASEURLS = {
     [constants_1.NetworkNames.MAINNET]: "https://explorer.qtum.org/insight-api",
-    [constants_1.NetworkNames.TESTNET]: "https://testnet.qtum.org/insight-api",
-    [constants_1.NetworkNames.REGTEST]: "http://localhost:3001/insight-api",
+    [constants_1.NetworkNames.TESTNET]: "https://mytestnet/insight-api",
+    [constants_1.NetworkNames.REGTEST]: "http://10.0.2.2:3001/insight-api",
 };
 class Insight {
     constructor(baseURL) {
